### PR TITLE
feat: UNT-T22247: Example app version update

### PR DESCRIPTION
**Pull request by nilesh-simform**
- Upgrade React Native Version
- Converted Js file to Tsx file

### CODE REVIEW

File: Example/android/app/proguard-rules.pro [hunk 1]
```diff
@@ -8,83 +8,3 @@
 #   http://developer.android.com/guide/developing/tools/proguard.html
 
 # Add any project specific keep options here:
-
```
Comment (Krunal-K-SimformSolutions) [inline Example/android/app/proguard-rules.pro:11]: Any specific resonance to remove guard

File: Example/App/BarSpinnerButton.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, {useCallback, useState} from 'react';
+import {StyleSheet, Text} from 'react-native';
+import SpinnerButton from 'react-native-spinner-button';
+import styles from './styles/AppStyles';
+
+const BarSpinnerButton = () => {
```
Comment (Krunal-K-SimformSolutions) [inline Example/App/BarSpinnerButton.tsx:6]: Add function component type

File: Example/App/BarSpinnerButton.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const [isLoading, setLoading] = useState(false);
```
Comment (Krunal-K-SimformSolutions) [inline Example/App/BarSpinnerButton.tsx:7]: Missing useState type

File: Example/App/BarSpinnerButton.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+  const handleButtonPress = useCallback(() => {
```
Comment (Krunal-K-SimformSolutions) [inline Example/App/BarSpinnerButton.tsx:9]: Missing type of useCallback

**Review by Krunal-K-SimformSolutions — COMMENTED** [review]
Missing all the example file in 
- Function Component type
- useState hook type
- useCallback hook type

File: Example/App/BarSpinnerButton.tsx (hunk 2 above)
Comment (nilesh-simform) [reply Example/App/BarSpinnerButton.tsx]: Added, Please Check

File: Example/App/BarSpinnerButton.tsx (hunk 3 above)
Comment (nilesh-simform) [reply Example/App/BarSpinnerButton.tsx]: Added, Please Check

File: Example/App/BarSpinnerButton.tsx (hunk 4 above)
Comment (nilesh-simform) [reply Example/App/BarSpinnerButton.tsx]: Added, Please Check

**Pull request merged**